_includes/release-assets.html

```diff
@@ -31,13 +31,31 @@
 </td>
 <td>
 	<ul class="row">
+		{% assign signed_firefox = include.assets | where_exp:"item", "item.name contains '-firefox.'" | first %}
+		{% assign unsigned_firefox = include.assets | where_exp:"item", "item.name contains '-firefox-unsigned'" | first %}
 
-		{% assign firefox = include.assets | where_exp:"item", "item.name contains '-firefox.'" | first %}
-		{% if firefox != nil %}
-			<li><a href="{{ firefox.browser_download_url}}" target="_blank">Firefox</a></li>
+		{% if include.latest %}
```

Comment:
I've managed to compact the logic a bit to avoid repeating code:
```
{% if signed_firefox != nil %}
	<li><a href="{{ signed_firefox.browser_download_url}}" target="_blank">Firefox</a></li>
{% elsif !include.latest && unsigned_firefox != nil %}
	<li><a href="{{ unsigned_firefox.browser_download_url}}" target="_blank">Firefox (unsigned)</a></li>
{% else %}
	<li style="text-decoration: line-through">Firefox</li>
{% endif %}

{% if include.latest && (signed_firefox == nil || unsigned_firefox != nil and unsigned_firefox.created_at > signed_firefox.created_at) %}
	<li><a href="{{ unsigned_firefox.browser_download_url}}" target="_blank">Firefox (unsigned) - More recent</a></li>
{% endif %}
```
Note that I never used Jekyll and I haven't tested it so some syntax might be wrong. Also my boolean logic might be flawed...

Reply:
Mostly correct, slightly changed to work correctly (and not &&, or not ||, and !include.latest is not a proper test for nil in Jekyll, so include.latest == nil instead).